packages/shared-action-utils/src/checkActionDistributable.ts

```diff
@@ -0,0 +1,31 @@
+import { getExecOutput } from "@actions/exec";
+import { getActions } from "./getActions";
+
+// Checks whether any actions have not had their distributable files updated
+async function checkActionDistributable(): Promise<void> {
+    const actions = await getActions();
+    let hasUncommittedChanges = false;
+
+    for (const action of actions) {
+        const { exitCode, stderr, stdout } = await getExecOutput("git", ["diff", "--name-only", action.directoryPath], {
```

Comment:
I just want to check my understanding here, this will return something if the folder has a change to what the Dev put in after the PR was created (e.g the forgot to run build), because we have run the build command in the workflow, Is that correct?

Reply:
Yep that's right, we run build as part of the workflow and if that is different to what was committed then the build fails.